etc/kayobe/stackhpc-overcloud-dib.yml

```diff
@@ -55,8 +55,10 @@ stackhpc_overcloud_dib_env_vars:
   # Avoid DNS queries during sudo commands, since we might not always have working DNS.
   DIB_SUDOERS_CONFIG: |
     Defaults  !fqdn
-  # FIXME: Support templating repo files.
-  # DIB_YUM_MINIMAL_BOOTSTRAP_REPOS: /path/to/dir/containing/dib-mirror-*.repo
+  # On Rocky Linux 9, use the host's repo files.
+  # These will have been configured to point to Test Pulp repo snapshots during the 'host configure' step.
+  # FIXME: This assumes we are building on RL9, which may not be the case.
```

Comment:
On yoga we use the CS8 builder runners: https://github.com/stackhpc/stackhpc-kayobe-config/blob/stackhpc/yoga/.github/workflows/overcloud-host-image-build.yml#L42
We do switch to RL9 for Zed though: https://github.com/stackhpc/stackhpc-kayobe-config/blob/stackhpc/zed/.github/workflows/overcloud-host-image-build.yml#L30

Reply:
Right - we could make it work for CS8 too, but thought I'd focus on the future :)